### PR TITLE
init crypto js project

### PR DESCRIPTION
**Pull request by DieMyst**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -45,15 +47,40 @@ lazy val `b-tree-client` = project.in(file("b-tree/client"))
...
+      circeCore,
+      circeParser
+    )
+  )
+  .jvmSettings(
+    //
+  )
+  .jsSettings(
+    libraryDependencies ++= Seq(
+      "org.scala-js" %%% "scalajs-dom" % "0.9.3"
```
Comment (alari) [inline build.sbt]: You don't need DOM in crypto, do you?

Comment (DieMyst) [reply build.sbt]: i need it for Web Crypto API
https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API

Comment (alari) [reply build.sbt]: Could you add this comment & link in the file, so that it's obvious for everyone?

File: build.sbt [hunk 2]
```diff
@@ -45,15 +47,43 @@ lazy val `b-tree-client` = project.in(file("b-tree/client"))
...
-lazy val `crypto` = project.in(file("crypto"))
+lazy val `crypto` = crossProject(JVMPlatform, JSPlatform)
+  .withoutSuffixFor(JVMPlatform)
+  .crossType(FluenceCrossType)
+  .settings(
+    commons,
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "cats-core" % Cats1V,
+      "org.scodec" %%% "scodec-bits" % ScodecV,
+      "org.scalatest" %%% "scalatest" % ScalatestV % Test,
```
Comment (alari) [inline build.sbt]: let's put test dependencies at the end?

File: crypto/jvm/src/test/scala/fluence.crypto/SignatureSpec.scala [hunk 3]
```diff
@@ -26,7 +26,7 @@ import fluence.crypto.keypair.KeyPair
 import org.scalatest.{ BeforeAndAfterAll, Matchers, WordSpec }
 import scodec.bits.ByteVector
 
-import scala.util.{ Random, Try }
+import scala.util.Random
```
Comment (alari) [inline crypto/jvm/src/test/scala/fluence.crypto/SignatureSpec.scala]: Folder name looks strange

**Pull request merged**